css/Components/FeedbackSlider.scss

```diff
@@ -0,0 +1,43 @@
+@import "../Theme/Color";
+
+$FeedbackSlider-dots-size: 9px;
+
+.FeedbackSlider-dots {
+  width: 100%;
+  margin: 0;
+
+  list-style: none;
+
+  li {
```

Comment:
The slider has an option to pass custom HTML class. This removes any existing styles and classes from child elements but it's better to avoid overrides and the mess that comes with it. To customise the child elements I do need to use the actual tags.